packages/modules-sdk/src/medusa-app.ts

```diff
@@ -112,6 +119,23 @@ async function initializeLinks(linkModules, injectedDependencies) {
   }
 }
 
+async function getMigrationsFromLinkModule(
+  linkModule: string
+): Promise<RunMigrationFn | undefined> {
+  try {
+    const { runMigrations } = await import(linkModule)
```

Comment:
comment: This is not webpack compatible due to the nature of the import that is fully dynamic. We can see that at a later point but just to keep in mind